remme/shared/utils.py

```diff
@@ -44,3 +44,10 @@ def message_to_dict(message):
         message,
         including_default_value_fields=True,
         preserving_proto_field_name=True)
+
```

Comment:
We already have a singleton implementation which can be used as a decorator https://github.com/Remmeauth/remme-core/blob/master/remme/shared/singleton.py